reviews/review-03+2020-11-30.md

```diff
@@ -0,0 +1,169 @@
+book. I don't seem to find Appendix E in my PDF version of the manuscript.
+
+f. Sec 1.3: I might add a sentence or two generalizing the Zipf's law
+example. If you want to "sell" the book, I think I would emphasize how looking
+for patterns in data is common to most scientific analyses and so you have
+chosen a straightforward example that will allow you to teach the concrete
+objectives you state. Rather than Syllabus, maybe something more direct---"What
+you will learn."
+
+g. Given what I've read up the Chapter 2 on the Unix shell, I am under the
+assumption that the audience of this book is already using Python at least an
+intermediate level. I'd like to believe that this level of user at least knows
+how to manipulate Linux directories. Everything in this chapter felt like
+prerequisite, so perhaps it should be an appendix?
+
+h. I think Chapter 3 content was more appropriate for an intermediate Python
+user who did not do much with the shell. I might make this 'the' Unix chapter
+with a little bit of introductory context as to why the content will be
+important for RSEs to be effective in that role.
+
+i. Should Sec 7.6 discuss using GitHub issues templates to create more effective
+bug reports.
+
+j. I think Docker is an essential reproducibility tool and feel that a chapter
+or section on the subject would be useful.
+
+k. The Finale at this point seems like an afterthought. Perhaps the great
+content on working within teams and communities could be reshaped into something
+that sends people off thinking about (and hopefully acting on) the helpful
+advice presented in these sections?
+
+l. Several exercise answers are missing; noting for thoroughness.
+
+m. Given the current title, I think some additional references to the history of
+the "research software engineer" term and the current and growing communities
+would be appropriate.
+
+4. Are the chapter titles/subtitles appropriate? If not, how can they be
+modified?
+
+a. I don't know if it would make a difference, but I wonder if the title of the
+book should be "Engineering Research Software with Python." I make this comment
+for three reasons: 1) putting the verb first seems more active; 2) the term RSE
+is still growing in notoriety so those who do not know it may not pick up the
+connection; 3) I think the author's want everyone to effectively use Python to
+develop their software even if they don't consider themselves RSEs; the current
+form might make it sound like you already need to be an RSE to use this book.
+
+b. See Q3 comment e about Sec 1.3: Syllabus seems like the wrong title in the
+context of a book.
+
+c. Noticing a trend of noun-heavy titles. Could Sec 1.4 have a verb---"Setting
+up your project"? The current title might make more sense to me deep inside an
+exercise. I am at the beginning and I think active titles are easier for me to
+consume (or make the choice that I want to spend time reading).
```

Comment:
All of our Chapters have active titles and most of the Sections do, but not all of the Sections. Should we aim for active titles for all Sections (besides template sections like Key Points and Exercises)?

Reply:
:-1: - I think what we have is good enough to ship.